themes/wilson_theme_starterkit/package.json

```diff
@@ -12,10 +12,11 @@
     "stylelint": "stylelint $npm_package_config_stylesSrc/**/**/*.scss --config .stylelintrc",
     "sass": "sass $npm_package_config_stylesSrc:$npm_package_config_stylesDist",
     "postcss": "postcss -r $npm_package_config_stylesDist/**/**/*.css",
+    "prettier": "prettier --write $npm_package_config_scriptsSrc",
     "eslint": "eslint $npm_package_config_scriptsSrc",
     "babel": "babel $npm_package_config_scriptsSrc --out-dir $npm_package_config_scriptsDist",
     "build:styles": "npm run stylelint && npm run sass && npm run postcss",
-    "build:scripts": "npm run eslint && npm run babel",
+    "build:scripts": "if-env NODE_ENV=production && npm run prettier && npm run eslint && npm run babel || npm run eslint && npm run babel",
```

Comment:
I didn't know you could do more complicated commands with ifs! 

so what will this do - "if the node_env is production then run prettier, eslint and babel, otherwise just run eslint and babel" ? That's very nifty.

Reply:
Ah I see there's a new package included below. Nice!